CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-## Unreleased
- - Adds retry mechanism when checkout Derby from SVN repository [#158](https://github.com/logstash-plugins/logstash-integration-jdbc/pull/158)
```

Comment:
I removed this from the changelog because it is build-chain related change that is not user-facing or part of the released artifacts.